Library/Homebrew/formula.rb

```diff
@@ -1810,6 +1818,17 @@ def mirror(val)
       define_method(type) { |val| stable.send(type, val) }
     end
 
+    # @!attribute [w] gpg
+    # An optional mechanism to verify the download with GnuPG as well as the
+    # standard checksum verification. Upstream must publish signature files
+    # for us to be able to use this mechanism. Those signature files often
+    # have either a ".gpg" or ".sig" file extension.
+    #
+    # <pre>gpg "https://ftpmirror.gnu.org/wget/wget-1.18.tar.xz.sig"</pre>
+    def gpg(val)
+      stable.gpg(val)
+    end
```

Comment:
I'm not planning to declare it safe day one & pour champagne everywhere, to reassure you. Current mindset is that there won't (for now) be a command line flag for it and you'll have to set both `HOMEBREW_DEVELOPER` and `HOMEBREW_ENABLE_GPG` for it to ever come into play. 

That will likely remain the situation for at least a while until I'm happy to call it safe & robust.

> Also may I suggest to use full fingerprint instead of any shorter version of it.

Yup.